helper/chunkify.js

```diff
@@ -0,0 +1,30 @@
+function chunkify(message, maxChars){
+    let lines = message.split(/\n/);
+    let chunks = [];
+    let numChars = 0;
+    for (let i = 0; i < lines.length; i++){
+        numChars = lines[i].length;
+        if (numChars < maxChars){
+            chunks.push(lines[i]);
+        } else {
+            let loc = lines[i].lastIndexOf(" ");
+            let subchunk = "";
+            let subsubchunks = [];
+            if (loc >= 0 && loc < maxChars){
+                chunks.push(lines[i].slice(0, loc));
+                subchunk = lines[i].slice(loc);

```

Comment:
I think you could change to `loc + 1`, so that the space is not added to the next subchunk (will cause an infinite recursion) 👍

Reply:
Resolved in latest fix, thanks Vishnu!